src/connection.ts

```diff
@@ -953,8 +953,9 @@ export class Connection extends EventEmitter {
             newTestMax = this.maximumPacketAmount.plus(this.testMaximumPacketAmount).dividedToIntegerBy(2)
             this.debug(`maximum packet amount is between ${this.testMaximumPacketAmount} and ${this.maximumPacketAmount}, trying: ${newTestMax}`)
           } else {
-            // Increase by 2 in this case since the amount to send and text max are equal which indicates the last packet was successful
-            newTestMax = this.testMaximumPacketAmount.times(2)
+            // Increase by 10% in this case since the amount to send and text max are equal which indicates the last packet was successful
+            const incrementBy = this.testMaximumPacketAmount.times(1.1).integerValue(BigNumber.ROUND_HALF_EVEN)
```

Comment:
Is there a reason for making it 1.1? Are you just trying to make it increase a little slower than when it's x2?

Reply:
Exactly, no real reason other than 10% being more reasonable than x2